automatic/git.install/tools/helpers.ps1

```diff
@@ -0,0 +1,64 @@
+function Get-InstallKey() {
+    $registryKeyName = 'Git_is1'
+    $installKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+    if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -eq 'true') {
```

Comment:
The logic in this seems wrong to me. But it is the same logic as in the original script [here](https://github.com/ferventcoder/chocolatey-packages/blob/master/automatic/git.install/tools/chocolateyInstall.ps1#L67). 

Shouldn't this be `if ((Get-ProcessorBits 32) -or $env:chocolateyForceX86 -eq 'true')`?

@ferventcoder Any idea about the logic behind this?